homeassistant/components/spotify/spotify_mappings.json

```diff
@@ -1,62 +1,62 @@
 {
   "clear-night": {
-    "warm": "Starry Night Vibes",
-    "cold": "Winter Star-gazing"
+    "warm": "Star Gazing ",
```

Comment:
Is this white-space intentional? It seems to make a difference when searching in spotify